gradle/init.d/dependencyUpdates.gradle

```diff
@@ -12,15 +12,20 @@ initscript {
   dependencies {
     //noinspection GradleDynamicVersion
     classpath "com.github.ben-manes:gradle-versions-plugin:+"
+    //noinspection GradleDynamicVersion
+    classpath "nl.littlerobots.version-catalog-update:nl.littlerobots.version-catalog-update.gradle.plugin:+"
   }
 }
 
 allprojects {
-  apply plugin: com.github.benmanes.gradle.versions.VersionsPlugin
+  if (it == rootProject) {
```

Comment:
In stead of this check you can change `allProjects` to `rootProject`

Reply:
Nice catch.